source/code-snippets/usage-examples/bulkWrite.ts

```diff
@@ -0,0 +1,53 @@
+import { MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const theaters = database.collection("theaters");
+
+    const result = await theaters.bulkWrite([
+      { insertOne:
+        {
+          "document": {
+            location: {
+              address: { street1: '3 Main St.', city: 'Anchorage', state: 'AK', zipcode: '99501' },
+            }
+          }
+        }
+      },
+      { insertOne:
+        {
+          "document": {
+            location: {
+              address: { street1: '75 Penn Plaza', city: 'New York', state: 'NY', zipcode: '10001' },
+            }
+          }
+        }
+      },
+      { updateMany:
+        {
+          "filter": { "location.address.zipcode" : "44011" },
```

Comment:
nit: Wanna run this through the formatter? the stringified keys are just not necessary

Reply:
Ran JS and TS examples through `prettier`!

Reply:
Nice! If you wanted those address objects to still be on one line you can increase the `--print-width` of prettier. But I think it looks fine either way

Reply:
Thanks for the tip! Will keep the `--print-width` flag in mind in the future